src/Microsoft.CodeAnalysis.Analyzers/Core/FixAnalyzers/FixerWithFixAllAnalyzer.cs

```diff
@@ -280,7 +280,8 @@ static bool IsViolatingCodeActionCreateInvocation(IInvocationOperation invocatio
                     IParameterSymbol param = invocation.TargetMethod.Parameters.FirstOrDefault(p => p.Name == EquivalenceKeyParameterName);
                     if (param == null)
```

Comment:
We could be checking that the overload contains the `nestedActions` param but I think that the current check is good enough.

Reply:
So the assumption here is CodeAction.Create will only have overloads without `equivalenceKey` parameter when not applicable? Seems reasonable.